clvm/operators.py

```diff
@@ -111,6 +111,10 @@ def default_unknown_op(op: bytes, args: CLVMObject) -> Tuple[int, CLVMObject]:
 
     cost_function = (op[-1] & 0b11000000) >> 6
     # the multiplier cannot be 0. it starts at 1
+
+    if len(op) > 5:
+        raise EvalError("invalid operator", args.to(op))
```

Comment:
We could just reset the cost in this case to something small, even a fixed value, rather than throw out the operator. I dunno, probably doesn't much matter. What do you think?

Reply:
I would think we have enough space for additional operators. If we feel we should have more space, I would think the lower bits (that are ignored for cost purposes) should be expanded.